R/backend-snowflake.R

```diff
@@ -37,3 +217,40 @@ simulate_snowflake <- function() simulate_dbi("Snowflake")
 # Link to full list: https://docs.snowflake.com/en/sql-reference/sql-all.html
 #' @export
 sql_table_analyze.Snowflake <- function(con, table, ...) {}
+
+snowflake_grepl <- function(pattern, x, ignore.case = FALSE, perl = FALSE, fixed = FALSE, useBytes = FALSE) {
+  # https://docs.snowflake.com/en/sql-reference/functions/regexp.html
+  if (any(c(perl, fixed, useBytes))) {
```

Comment:
I'd write `perl || fixed || useBytes` just to reinforce that these should be scalars.